politeiawww/proposals_test.go

```diff
@@ -457,50 +495,70 @@ func TestValidateProposal(t *testing.T) {
 				ErrorCode: www.ErrorStatusInvalidSignature,
 			}},
 
-		{"incorrect signature", *propBadSig, usr,
+		{"invalid public key", *propInvalidPubKey, usr,
 			www.UserError{
-				ErrorCode: www.ErrorStatusInvalidSignature,
+				ErrorCode: www.ErrorStatusInvalidPublicKey,
 			}},
 
 		{"no files", *propNoFiles, usr,
 			www.UserError{
-				ErrorCode: www.ErrorStatusProposalMissingFiles,
+				ErrorCode: www.ErrorStatusNoIndexFile,
```

Comment:
You changed the error message here but not the test name.  This is testing for "no index file" now.

Reply:
Ignore this comment.